src/controls/TeamChannelPicker/TeamChannelPicker.tsx

```diff
@@ -207,35 +214,46 @@ export const TeamChannelPicker: React.FunctionComponent<ITeamChannelPickerProps>
         return null;
       }
     },
-    [state.selectedTeamsChannels, props.onSelectedChannels]
+    [
+      state.selectedTeamsChannels,
+      props.onSelectedChannels,
+      themeVariant,
+      renderIconButtonRemoveStyles,
+      renderItemStylesMulti,
+      renderItemStylesSingle,
+    ]
   );
 
   // Render  control
   return (
-    <div style={{ width: "100%" }}>
-      {props.label && <Label>{props.label}</Label>}
-      <TagPicker
-        styles={
-          styles ??
-          (itemLimit && itemLimit > 1 ? pickerStylesMulti : pickerStylesSingle)
-        }
-        selectedItems={state.selectedTeamsChannels}
-        onRenderItem={_onRenderItem}
-        onRenderSuggestionsItem={_onRenderSuggestionsItem}
-        ref={picker}
-        onResolveSuggestions={useFilterSuggestedTeamsChannels}
-        getTextFromItem={getTextFromItem}
-        pickerSuggestionsProps={pickerSuggestionsProps}
-        onEmptyResolveSuggestions={(selectTeams) => {
-          return useFilterSuggestedTeamsChannels("", selectTeams);
-        }}
-        itemLimit={itemLimit ?? undefined}
-        onChange={(items) => {
-          dispatch({ type: "UPDATE_SELECTEITEM", payload: items });
-          onSelectedChannels(items);
-        }}
-        componentRef={picker}
-      />
-    </div>
+    <Customizer settings={{ theme: props.themeVariant }}>
+      <div style={{ width: "100%" }}>
+        {props.label && <Label>{props.label}</Label>}
+        <TagPicker
+          styles={
+            styles ??
+            (itemLimit && itemLimit > 1
+              ? pickerStylesMulti
+              : pickerStylesSingle)
+          }
+          selectedItems={state.selectedTeamsChannels}
+          onRenderItem={_onRenderItem}
+          onRenderSuggestionsItem={_onRenderSuggestionsItem}
+          ref={picker}
+          onResolveSuggestions={useFilterSuggestedTeamsChannels}
+          getTextFromItem={getTextFromItem}
+          pickerSuggestionsProps={pickerSuggestionsProps}
+          onEmptyResolveSuggestions={(selectTeams) => {
+            return useFilterSuggestedTeamsChannels("", selectTeams);
+          }}
+          itemLimit={itemLimit ?? undefined}
+          onChange={(items) => {
+            dispatch({ type: "UPDATE_SELECTEITEM", payload: items });
```

Comment:
Minor thing, but should 'UPDATE_SELECTEITEM' be 'UPDATE_SELECTEDITEM'? (added an extra "d")

Reply:
done